src/base-element.js

```diff
@@ -529,11 +531,13 @@ export class BaseElement {
    * may refuse to make a change in which case it will show the element's
```

Comment:
I'm still confused. We don't want ads to ever show the overflow in the collapse case. Is this comment correct?